README.md

```diff
@@ -1,6 +1,10 @@
 **This is ALPHA Software**
 
-This was built as a prototype to evaluate using react inside of our Ember apps. We are not yet using it in production. PRs and constructive questions and comments via [GitHub issues](https://github.com/AltSchool/ember-cli-react/issues/new) are highly encouraged.
+This was built as a prototype to evaluate using react inside of our Ember apps.
```

Comment:
Most of the changes were caused by Prettier (apparently I missed this file last time)